kernel/posix/pthread_common.c

```diff
@@ -18,3 +19,34 @@ void ready_one_thread(_wait_q_t *wq)
 		_ready_thread(th);
 	}
 }
+
+s64_t calculate_timeout(const struct timespec *abstime)
```

Comment:
`calculate_timeout()` is a bad function name for something that's so generic, especially if it's not a static function. Consider naming it `timespec_to_ms()` or something along these lines.

Reply:
sure i will change the name.